workflow-service-sdk/pom.xml

```diff
@@ -97,7 +136,8 @@
                                 <licenseName>The Apache Software License, Version 2.0</licenseName>
                                 <licenseUrl>http://www.apache.org/licenses/LICENSE-2.0.txt</licenseUrl>
                                 <scmConnection>scm:git:git://github.com/redhat-developer/parodos.git</scmConnection>
-                                <scmDeveloperConnection>scm:git:ssh://github.com/redhat-developer/parodos.git</scmDeveloperConnection>
+                                <scmDeveloperConnection>scm:git:ssh://github.com/redhat-developer/parodos.git
```

Comment:
We changed the organization. The URL is not correct anymore.

Reply:
Since the issue is not only related to this PR, I addressed your comment in a separate PR #180 .
I'll update the description of this PR and I'll add the /hold label to prevent merging .